js/binance.js

```diff
@@ -2598,7 +2598,7 @@ module.exports = class binance extends Exchange {
                 priceString = Precise.stringDiv (costString, amountString);
             }
         }
-        const id = undefined;
+        const id = orderId;
```

Comment:
The `id` of a trade must be unique, if we set `id = orderId`, that will result in having multiple trades with the same id.